File: exercicio.cs
class Aluno{
  public static Alunos = new List<Aluno>();
  public string Nome{get;set;}
  public string Telefone{get;set;}
  public float ValorHora{get;set;}

  public void Salvar(){
  	Aluno.Alunos.add(this);
  }

  public void Listar(){
  	foreach(var aluno in Aluno.Alunos){
  		Console.WriteLine("Nome: " + aluno.nome);
  	}
  }
}

File: exercicio.js
 var readline = require('readline-sync');

alunos = [];  
for(i=0;i<1;i++){    
  aluno = {}
  aluno.nome =  readline.question("Nome do Aluno: ");    
  aluno.telefone = readline.question("Telefone: ");     
  aluno.valor_hora = readline.question("Qual valor/hora: "); 

  alunos.push(aluno);  
} 

for(i=0;i<alunos.length;i++){
  console.log("Nome: " + alunos[i].nome); 
  console.log("Telefone: " + alunos[i].telefone); 
  console.log("Valor hora: " + alunos[i].valor_hora); 
}  